src/components/exchange/AvailableNetworksv2.tsx

```diff
@@ -0,0 +1,233 @@
+import lang from 'i18n-js';
+import React, { useCallback, useMemo } from 'react';
+import RadialGradient from 'react-native-radial-gradient';
+import Divider from '../Divider';
+import { CoinIcon } from '../coin-icon';
+import ChainBadge from '../coin-icon/ChainBadge';
+import { Box, Inline, Text } from '@/design-system';
+import networkInfo from '@/helpers/networkInfo';
+import { useNavigation } from '@/navigation';
+import { ETH_ADDRESS, ETH_SYMBOL } from '@/references';
+import Routes from '@/navigation/routesNames';
+import { position } from '@/styles';
+import { ethereumUtils } from '@/utils';
+import { CurrencySelectionTypes, ExchangeModalTypes, Network } from '@/helpers';
+import { useSwapCurrencyHandlers } from '@/hooks';
+import { AssetType, RainbowToken } from '@/entities';
+import { useTheme } from '@/theme';
+import { EthereumAddress } from '@rainbow-me/swaps';
+import { ButtonPressAnimation } from '../animations';
+import ContextMenuButton from '@/components/native-context-menu/contextMenu';
+
+const NOOP = () => null;
+
+type implementation = { address: EthereumAddress; decimals: number };
+
+const AvailableNetworksv2 = ({
+  asset,
+  networks,
+  hideDivider,
+  marginHorizontal = 19,
+}: {
+  asset: RainbowToken;
+  networks: Record<number, implementation>;
+  hideDivider: boolean;
+  marginHorizontal: number;
+}) => {
+  const { colors } = useTheme();
+  const { goBack, navigate } = useNavigation();
+
+  const radialGradientProps = {
+    center: [0, 1],
+    colors: colors.gradients.lightGreyWhite,
+    pointerEvents: 'none',
+    style: {
+      ...position.coverAsObject,
+      overflow: 'hidden',
+    },
+  };
+
+  const availableNetworks = useMemo(() => {
+    // we dont want to show mainnet
+    let newNetworks = networks;
+    delete newNetworks['1'];
+    return Object.keys(newNetworks).map(network =>
+      ethereumUtils.getNetworkFromChainId(Number(network))
+    );
```

Comment:
could we have something like

```
const newNetworks = Object.keys(network).filter(network => network !== Network.mainnet)
```

instead of deleting a key from the object